fre/make/create_docker_script.py

```diff
@@ -1,22 +1,34 @@
 '''
-TODO- make docstring
+Create a Dockerfile and container build script, and generate a model container.
```

Comment:
Creates a Dockerfile and an accompanying  build script?  (I'm thinking the word container is unecessary here since it's implied with the word Dockerfile... but if you think it's necessary, maybe the word image is better than container)
generates